### PR TITLE
[release/3.1] Add CPD strict dependencies for aspnetcore-tooling

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -70,6 +70,10 @@
       <Uri>https://github.com/dotnet/core-setup</Uri>
       <Sha>d0bc85c7c6047fcf43a99ea5f02da5136c118cce</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.Extensions.DependencyModel" Version="3.1.6">
+      <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-core-setup</Uri>
+      <Sha>3acd9b0cd16596bad450c82be08780875a73c05c</Sha>
+    </Dependency>
```
Comment (dougbu) [inline eng/Version.Details.xml:76]: dotnet/efcore needs this plus Microsoft.DotNet.PlatformAbstractions. Is it a problem that's not mentioned in dotnet/core-setup❔

Comment (mmitche) [reply eng/Version.Details.xml:76]: Yeah I only added them in context of aspnetcore-tooling. There are a LOT more to add.

Comment (wtgodbe) [reply eng/Version.Details.xml:76]: These packages come from core-setup, we don't need to mention them there. But we will need to do the same in EFCore

Comment (dougbu) [reply eng/Version.Details.xml:76]: @wtgodbe I bet the only remaining thing to get versions working in dotnet/efcore and dotnet/aspnetcore is to add a reference to Microsoft.DotNet.PlatformAbstractions in dotnet/extensions. Right @mmitche❔

Comment (dougbu) [reply eng/Version.Details.xml:76]: (If yes, I'll do the needful.)

Comment (mmitche) [reply eng/Version.Details.xml:76]: haven't checked efcore. For aspnetcore...it's much much more:

```
Checking for coherency updates...
Coherency updates failed for the following dependencies:
  Unable to update Microsoft.AspNetCore.Analyzer.Testing to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.AspNetCore.Analyzer.Testing
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.AspNetCore.BenchmarkRunner.Sources to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.AspNetCore.BenchmarkRunner.Sources
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.ActivatorUtilities.Sources to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.ActivatorUtilities.Sources
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Caching.Abstractions to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Caching.Abstractions
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Caching.SqlServer to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Caching.SqlServer
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Caching.StackExchangeRedis to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Caching.StackExchangeRedis
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.CommandLineUtils.Sources to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.CommandLineUtils.Sources
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Configuration.Abstractions to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Configuration.Abstractions
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Configuration.AzureKeyVault to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Configuration.AzureKeyVault
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Configuration.Binder to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Configuration.Binder
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Configuration.CommandLine to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Configuration.CommandLine
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Configuration.FileExtensions to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Configuration.FileExtensions
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Configuration.Ini to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Configuration.Ini
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Configuration.KeyPerFile to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Configuration.KeyPerFile
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Configuration.UserSecrets to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Configuration.UserSecrets
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Configuration.Xml to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Configuration.Xml
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.DependencyInjection.Abstractions to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.DependencyInjection.Abstractions
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.DiagnosticAdapter to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.DiagnosticAdapter
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Diagnostics.HealthChecks.Abstractions to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Diagnostics.HealthChecks.Abstractions
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Diagnostics.HealthChecks to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Diagnostics.HealthChecks
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.FileProviders.Abstractions to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.FileProviders.Abstractions
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.FileProviders.Composite to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.FileProviders.Composite
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.FileProviders.Embedded to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.FileProviders.Embedded
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.FileProviders.Physical to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.FileProviders.Physical
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.FileSystemGlobbing to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.FileSystemGlobbing
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.HashCodeCombiner.Sources to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.HashCodeCombiner.Sources
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Hosting.Abstractions to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Hosting.Abstractions
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Hosting to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Hosting
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Http to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Http
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Localization.Abstractions to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Localization.Abstractions
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Localization to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Localization
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Logging.Abstractions to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Logging.Abstractions
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Logging.AzureAppServices to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Logging.AzureAppServices
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Logging.Configuration to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Logging.Configuration
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Logging.Console to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Logging.Console
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Logging.Debug to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Logging.Debug
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Logging.EventSource to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Logging.EventSource
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Logging.EventLog to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Logging.EventLog
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Logging.TraceSource to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Logging.TraceSource
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Logging.Testing to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Logging.Testing
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.ObjectPool to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.ObjectPool
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Options.ConfigurationExtensions to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Options.ConfigurationExtensions
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Options.DataAnnotations to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Options.DataAnnotations
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Options to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Options
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.ParameterDefaultValue.Sources to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.ParameterDefaultValue.Sources
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.Primitives to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.Primitives
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.TypeNameHelper.Sources to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.TypeNameHelper.Sources
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.ValueStopwatch.Sources to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.ValueStopwatch.Sources
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.WebEncoders to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.Extensions.WebEncoders
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.JSInterop to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.JSInterop
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Bcl.AsyncInterfaces to have coherency with Microsoft.NETCore.App.Runtime.win-x64: https://github.com/dotnet/core-setup @ 9a599ffefec5df0619e5154a8e5c14eba09dc5e6 does not contain dependency Microsoft.Bcl.AsyncInterfaces
    - Add the dependency to https://github.com/dotnet/core-setup.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.CSharp to have coherency with Microsoft.NETCore.App.Runtime.win-x64: https://github.com/dotnet/core-setup @ 9a599ffefec5df0619e5154a8e5c14eba09dc5e6 does not contain dependency Microsoft.CSharp
    - Add the dependency to https://github.com/dotnet/core-setup.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update System.ComponentModel.Annotations to have coherency with Microsoft.NETCore.App.Runtime.win-x64: https://github.com/dotnet/core-setup @ 9a599ffefec5df0619e5154a8e5c14eba09dc5e6 does not contain dependency System.ComponentModel.Annotations
    - Add the dependency to https://github.com/dotnet/core-setup.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update System.Net.Http.WinHttpHandler to have coherency with Microsoft.NETCore.App.Runtime.win-x64: https://github.com/dotnet/core-setup @ 9a599ffefec5df0619e5154a8e5c14eba09dc5e6 does not contain dependency System.Net.Http.WinHttpHandler
    - Add the dependency to https://github.com/dotnet/core-setup.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update System.Net.WebSockets.WebSocketProtocol to have coherency with Microsoft.NETCore.App.Runtime.win-x64: https://github.com/dotnet/core-setup @ 9a599ffefec5df0619e5154a8e5c14eba09dc5e6 does not contain dependency System.Net.WebSockets.WebSocketProtocol
    - Add the dependency to https://github.com/dotnet/core-setup.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update System.Reflection.Metadata to have coherency with Microsoft.NETCore.App.Runtime.win-x64: https://github.com/dotnet/core-setup @ 9a599ffefec5df0619e5154a8e5c14eba09dc5e6 does not contain dependency System.Reflection.Metadata
    - Add the dependency to https://github.com/dotnet/core-setup.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update System.ServiceProcess.ServiceController to have coherency with Microsoft.NETCore.App.Runtime.win-x64: https://github.com/dotnet/core-setup @ 9a599ffefec5df0619e5154a8e5c14eba09dc5e6 does not contain dependency System.ServiceProcess.ServiceController
    - Add the dependency to https://github.com/dotnet/core-setup.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update System.Threading.Channels to have coherency with Microsoft.NETCore.App.Runtime.win-x64: https://github.com/dotnet/core-setup @ 9a599ffefec5df0619e5154a8e5c14eba09dc5e6 does not contain dependency System.Threading.Channels
    - Add the dependency to https://github.com/dotnet/core-setup.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.Extensions.DependencyModel to have coherency with Microsoft.Extensions.Logging: https://github.com/dotnet/extensions @ c633aee8cb9b04369d78e57ad6548d6369215447 does not contain dependency Microsoft.Extensions.DependencyModel
    - Add the dependency to https://github.com/dotnet/extensions.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Internal.AspNetCore.Analyzers to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Internal.AspNetCore.Analyzers
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
  Unable to update Microsoft.AspNetCore.Testing to have coherency with Microsoft.EntityFrameworkCore: https://github.com/dotnet/efcore @ 85061a017ee445ef7c8d49b7256e16881aa19db8 does not contain dependency Microsoft.AspNetCore.Testing
    - Add the dependency to https://github.com/dotnet/efcore.
    - Pin the dependenency.
    - Remove the CoherentParentDependency attribute.
Error: Failed to update coherent parent tied dependencies.
```

**Pull request merged**